azure-pipelines.yml

```diff
@@ -296,6 +296,14 @@ jobs:
         testGroup: outerloop
         readyToRun: true
 
+# Format
+- template: eng/platform-matrix.yml
+  parameters:
+    jobTemplate: format-job.yml
+    platforms:
+    - Windows_NT_x64
+    - Linux_x64
```

Comment:
Can you please flip Windows_NT_x64 and Linux_x64 to keep the order consistent with default values for platforms in platform-matrix.yml